lua/lspconfig/lelwel_ls.lua

```diff
@@ -0,0 +1,24 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.lelwel_ls = {
+  default_config = {
+    cmd = { 'lelwel-ls' },
+    filetypes = { 'llw' },
+    root_dir = function(fname)
+      return util.find_git_ancestor(fname) or util.path.dirname(fname)
```

Comment:
You can remove util.path.dirname now, it's handled automatically.

root_dir = util.find_git_ancestor